src/explore-education-statistics-admin/src/pages/release/data/components/__tests__/ReleaseDataUploadsSection.test.tsx

```diff
@@ -24,6 +24,7 @@ const releaseDataFileService = _releaseDataFileService as jest.Mocked<
 describe('ReleaseDataUploadsSection', () => {
   const testDataFiles: DataFile[] = [
     {
+      id: '52d30125-6381-455a-abc6-eab82c943fc9',
```

Comment:
Rather than use actual uuids for frontend tests, I'd prefer the tests keep it simple with ids like `file-1`, `file-2`, etc. Makes it easier to type out when you're doing assertions on them.

Reply:
Thanks I've done this now